jobs.py

```diff
@@ -10,6 +10,8 @@
 from app.constants.response_privacy import PRIVATE
 from app.lib.db_utils import update_object, create_object
 from app.lib.email_utils import send_email
+from io import StringIO
+import traceback
```

Comment:
no need for StringIO (see below) and can you move the traceback import to the top please?